<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mwGrapher</class>
 <widget class="QMainWindow" name="mwGrapher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="vlTree">
        <item>
         <widget class="QTreeWidget" name="twTree">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabGraph">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="mToolsBarOrientation">
     <property name="title">
      <string>Tools Bar Orientation</string>
     </property>
     <addaction name="miNorth"/>
     <addaction name="miSouth"/>
     <addaction name="miWest"/>
     <addaction name="miEast"/>
    </widget>
    <addaction name="mToolsBarOrientation"/>
   </widget>
   <widget class="QMenu" name="mHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="mEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="miAddGraphZone"/>
    <addaction name="miConnectNodes"/>
   </widget>
   <addaction name="mEdit"/>
   <addaction name="mWindow"/>
   <addaction name="mHelp"/>
  </widget>
  <widget class="QToolBar" name="tbTools">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="miNorth2">
   <property name="text">
    <string>North</string>
   </property>
  </action>
  <action name="miSouth2">
   <property name="text">
    <string>South</string>
   </property>
  </action>
  <action name="miWest2">
   <property name="text">
    <string>West</string>
   </property>
  </action>
  <action name="miEast2">
   <property name="text">
    <string>East</string>
   </property>
  </action>
  <action name="miConnectNodes">
   <property name="text">
    <string>Connect Nodes</string>
   </property>
  </action>
  <action name="miNorth">
   <property name="text">
    <string>North</string>
   </property>
  </action>
  <action name="miSouth">
   <property name="text">
    <string>South</string>
   </property>
  </action>
  <action name="miWest">
   <property name="text">
    <string>West</string>
   </property>
  </action>
  <action name="miEast">
   <property name="text">
    <string>East</string>
   </property>
  </action>
  <action name="miAddGraphZone">
   <property name="text">
    <string>Add GraphZone</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
